common/djangoapps/student/admin.py

```diff
@@ -187,6 +232,5 @@ class Meta(object):
 admin.site.register(LogoutViewConfiguration, ConfigurationModelAdmin)
 admin.site.register(RegistrationCookieConfiguration, ConfigurationModelAdmin)
 
-
```

Comment:
is it necessary?

Reply:
no, but it was contradicting python code style and since I'm here and it took less then a second to fix and it does not need additional testing I'd vote this change to stay